cli/build.rs

```diff
@@ -0,0 +1,21 @@
+use std::process::Command;
+
+fn main() {
+    let hash = Command::new("git")
```

Comment:
this is going to be flaky and will break things. it is better to have it in automated builds. build.rs is useful e.g. with protoc for code generation but not for managing build metadata.